stdlib/src/builtin/debug_assert.mojo

```diff
@@ -23,7 +23,9 @@ from builtin._location import __call_location, _SourceLocation
 
 
 @always_inline
-fn debug_assert[stringable: Stringable](cond: Bool, msg: stringable):
+fn debug_assert[
+    *stringable: Stringable
+](cond: Bool, *message_parts: *stringable, sep: StringLiteral = " "):
```

Comment:
Please don't use `StringLiteral` for runtime arguments. We will have `StaticString` soon, so you can wait for that, or move this to a parameter.

Reply:
Or better yet, just don't have this functionality. Users can manually add spaces if they really need to.